components/tools/OmeroPy/src/omero/cli.py

```diff
@@ -1567,6 +1568,10 @@ def __call__(self, arg):
             for id in parts[1].split(","):
                 if "-" in id:
                     needsForce = True
+                    warnings.warn("Using '--dry-run'. \
```

Comment:
This doesn't yet take into account whether the user passed the flag or not, i.e. it will be shown even if they passed `--force`